core/include/gnuradio-4.0/Block.hpp

```diff
@@ -206,7 +211,7 @@ concept BlockLike = requires(T t, std::size_t requested_work) {
     requires !std::is_copy_assignable_v<T>;
     // requires !std::is_move_constructible_v<T>;
     // requires !std::is_move_assignable_v<T>;
-};
```

Comment:
Is this check obsolete now `{ t.work(requested_work) } -> std::same_as<work::Result>;`? (line 204 in updated file, sorry I could not + at this line since it was not changed)